src/components/status-panel/index.js

```diff
@@ -30,11 +30,11 @@ class StatusPanel extends Component {
 				</div>
 			),
 			intermediary: (
-				<p>We're building your environment right now, just sit back and we'll let you know when it's all ready.</p>
+				<p>We&quot;re building your environment right now, just sit back and we&quot;ll let you know when it&quot;s all ready.</p>
```

Comment:
```suggestion
				<p>We&#8217;re building your environment right now, just sit back and we&8217;ll let you know when it&8217;s all ready.</p>
```

WordPress uses curly apostrophes. 🙂